haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -7,7 +7,7 @@ LICENSE="GNU GPL v2"
 REVISION="1"
 gitSrcRevision="e3d2c76f0645076f4e05ea942427e0f6e919948b"
 SOURCE_URI="https://github.com/grahams/alivejournal/archive/$gitSrcRevision.tar.gz"
-CHECKSUM_SHA256="198950a11a0cb38cd9093d08619991ad6bbd2d376e7b1999b09a8a556d9add68"
+CHECKSUM_SHA256="8215d4ec005fd345c15ece8223f022cc6d434d940dc3c699a00c87c4e8b6ac49"
```

Comment:
I've tried the updated recipe but I still get the same error as with the previous patch:
```
sources/alivejournal-e3d2c76f0645076f4e05ea942427e0f6e919948b/src/LJApplication/LJAccount/LJViewHistory/LJViewHistory.h:37: syntax error before `*'
make: *** [/boot/system/develop/etc/makefile-engine:291: objects/LJAccount.o] Error 1
```
Well, at least this means the patch was correctly upstreamed :-)
I still have 4 sources that compile without any error, but the compilation fails with the fifth file:
```
cd haiku-apps/alivejournal/work-1.61/sources/alivejournal-e3d2c76*
find . -name "*.o" | wc -l ; find . -name "*.cpp" | wc -l
4
23
```
As expected, I also get the same error if I update to the latest gitSrcRevision:
```diff
-gitSrcRevision="e3d2c76f0645076f4e05ea942427e0f6e919948b"
+gitSrcRevision="cbd31ce1bdb61a58053d25eaca8e275efcf464ec"
 SOURCE_URI="https://github.com/grahams/alivejournal/archive/$gitSrcRevision.tar.gz"
-CHECKSUM_SHA256="8215d4ec005fd345c15ece8223f022cc6d434d940dc3c699a00c87c4e8b6ac49"
+CHECKSUM_SHA256="e4f8a66660ae3e89d6d9a8993f35f8912c28a3fee72896288f61f611d74998ad"
```